localization.json

```diff
@@ -1,248 +1,248 @@
 {
-    "en": {
```

Comment:
Please keep tabs at 4 spaces

Reply:
Thats not for me to decide, you can manually merge if you want